src/EventManager.php

```diff
@@ -174,8 +164,10 @@ public function addIdentifiers($identifiers)
      * @return ResponseCollection All listener return values
      * @throws Exception\InvalidCallbackException
      */
-    public function trigger($event, $target = null, $argv = [], $callback = null)
+    public function trigger($event, $target = null, $argv = array(), callable $callback = null)
```

Comment:
I'll run it through the new CS rules when complete; right now, I'm mainly
testing some concepts for refactoring.
On Jul 3, 2015 4:50 PM, "Abdul Malik Ikhsan" <notifications@github.com>
wrote:

> In src/EventManager.php
> <https://github.com/zendframework/zend-eventmanager/pull/4#discussion_r33881585>
> :
>
> > @@ -174,8 +164,10 @@ public function addIdentifiers($identifiers)
> >       * @return ResponseCollection All listener return values
> >       * @throws Exception\InvalidCallbackException
> >       */
> > -    public function trigger($event, $target = null, $argv = [], $callback = null)
> > +    public function trigger($event, $target = null, $argv = array(), callable $callback = null)
>
> $argv = array() can be $argv = []
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-eventmanager/pull/4/files#r33881585>
> .
>